schemas/v1/14_vm_messages_migration.go

```diff
@@ -0,0 +1,21 @@
+package v1
+
+func init() {
+	patches.Register(
+		14,
+		`
+{{- if and .SchemaName (ne .SchemaName "public") }}
+SET search_path TO {{ .SchemaName }},public;
+{{- end }}
+
+SELECT create_hypertable(
+	'vm_messages',
+	'height',
+	chunk_time_interval => 2880,
```

Comment:
What is the significance of 2880? The number of epochs in a day?

Reply:
Yes, number of epochs in a day. So with the retention policy, we can drop a chunk everyday.

Reply:
can you add a comment stating this please?